lang/ui.en.json

```diff
@@ -411,6 +411,10 @@
     "defaultMessage": "Short collections of movement data lasting 1 second. Your data samples are only stored on your computer, they do not get sent to anyone else.",
     "description": "Tooltip for data samples heading"
   },
+  "default-project-name": {
+    "defaultMessage": "Translated Untitled",
```

Comment:
To be changed back to Untitled once ensured working

Reply:
[71b52fa](https://github.com/microbit-foundation/ml-trainer/pull/462/commits/71b52fa8bf965b9f5e2159fe6850dfd19ebd0633)